### DISCUSSION TITLE
Welcome to Vumbnail Discussions!

### DISCUSSION THREAD

**Discussion by ThatGuySam** [ThatGuySam/vimeo-thumbnail-server#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by trashbrah** [comment]
hello, how's your embed project coming along? I have a large video hosted on vimeo it's 149mb and it's destroying load speed, what's my best bet ?

**Reply by ThatGuySam** [reply]
@trashbrah  Can you share the url?
